composer.json

```diff
@@ -1,12 +1,17 @@
 {
     "name": "friendsofsymfony/message-bundle",
     "type": "symfony-bundle",
-    "description": "Provides user-to-user messaging features for your Symfony2 application.",
+    "description": "Provides user-to-user messaging features for your Symfony application.",
     "keywords": ["symfony", "message", "messaging"],
     "homepage": "https://github.com/FriendsOfSymfony/FOSMessageBundle",
     "license": "MIT",
     "authors": [
         {
+            "name": "Titouan Galopin",
```

Comment:
While I agree with the fact that most contributions fall under the Friends of Symfony author, I thought the "author" section refered to who people should contact when they have an issue (ie. the maintainer). I receive mails of people explaining a problem they have with the bundle but also saying they are not sure I'm the official author. If you think that's a problem, I can remove it, no problem :) (or put it in the README perhaps?).